modules/javafx.graphics/src/main/java/javafx/scene/robot/Robot.java

```diff
@@ -0,0 +1,373 @@
+/*
```

Comment:
BTW the api for automatic delays between the actions can be useful in this class

Reply:
Which API is that?

Reply:
I guess he is referring to [java.awt.Robot::setAutoDelay](https://docs.oracle.com/javase/10/docs/api/java/awt/Robot.html#setAutoDelay(int)). While it could be useful, it seems out of scope for openjfx 11, given the threading issues we would need to consider.